README.md

```diff
@@ -107,6 +107,9 @@ proxy requests. The following options are supported:
   Example: `["cookie"]`
 * dictionary of lowercase strings `setHeaders` - Set headers for the request (overwrites existing ones).  
   Example: `{"x-powered-by": "CORS Anywhere"}`
+* number `corsMaxAge` - If set, an Access-Control-Max-Age request header with this value (in seconds) will be added.
+  Defaults to 0.  
```

Comment:
This line can be removed `"If set, ..." implies that the header is not set if not specified, which is already clear.